mappings/net/minecraft/structure/RuinedPortalFeaturePiece.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_5189 net/minecraft/world/gen/feature/RuinedPortalFeaturePiece
+CLASS net/minecraft/class_5189 net/minecraft/structure/RuinedPortalFeaturePiece
```

Comment:
Maybe `RuinedPortalStructurePiece` since it extends `StructurePiece`?

Reply:
Yeah , done